src/ui/app/redux/slices/transactions/index.ts

```diff
@@ -154,6 +154,11 @@ const slice = createSlice({
     reducers: {},
     extraReducers: (builder) => {
         builder.addCase(sendTokens.fulfilled, (state, { payload }) => {
+            // This line of code was giving me "TS2589: Type instantiation is excessively deep and possibly infinite."
```

Comment:
This is the only line of this PR that I'm a bit unsure of. @0xNeon-opensource or @jaredcosulich do you see this error in your IDE or at runtime ever?

Reply:
Yes, I see it intermittently and I'm not sure why